ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -148,25 +147,31 @@ ompi_mtl_ofi_context_progress(int ctxt_id)
                  * thread fetches the entry while others get -FI_EAGAIN
                  * indicating an empty queue, which is not erroneous.
                  */
-                if (ret == -FI_EAGAIN)
+                if (ret == -FI_EAGAIN) {
                     return count;
+                }
                 opal_output(0, "%s:%d: Error returned from fi_cq_readerr: %s(%zd).\n"
                                "*** The Open MPI OFI MTL is aborting the MPI job (via exit(3)).\n",
                                __FILE__, __LINE__, fi_strerror(-ret), ret);
-                fflush(stderr);
-                exit(1);
+                goto bail;
+            }
+
+            if (!error.op_context) {
+                opal_output(0, "%s:%d: Error returned from fi_cq_readerr with null context. "
```

Comment:
All of these `opal_output()` calls really should be show_help calls -- show_help is de-duplicated at `mpirun`.

Reply:
I don't know if this error will cause a lot of duplication - it should be rather rare and isolated.

So what is the intended use case for opal_output?

Reply:
I don't think show_help is needed here.  Certainly wouldn't be wrong, but the current code base (including this PR) is full of existing calls to opal_output() for errors that are unlikely to be correlated across nodes.